web/html/src/components/dialog/Dialog.css

```diff
@@ -0,0 +1,17 @@
+:global(.modal-body){
+    height: auto;
+    overflow-y: auto;
+}
+
+:global(.modal-overlay){
+  position: fixed;
+  inset: 0px;
+  background-color: rgba(0, 0, 0, 0.5);
+  z-index: 1040;
```

Comment:
Similar to previous comments about z-index, having arbitrary fixed values will get hard to manage quickly. Perhaps moving z-index declarations to a shared variable might help, what do you think @ncounter?

Reply:
indeed, a centralized place for them would be nice

Reply:
I think something along the lines of this would be handy: https://css-tricks.com/handling-z-index/#maps-might-be-cleaner

Reply:
Found out that I could just reuse bootstrap's zindex variable in the less files... no map for this time :grin: